package.json

```diff
@@ -42,29 +42,28 @@
     "co-wait": "0.0.0",
     "docker-raw-stream": "^1.3.1",
     "dockerode": "^2.0.4",
-    "eslint-config-probo": "^1.0.2",
+    "eslint-config-probo": "^2.0.0",
     "event-stream": "^3.3.2",
     "express": "^4.12.3",
     "flake-idgen": "^1.0.0",
     "github": "^0.2.3",
-    "github-api": "^0.10.3",
```

Comment:
It turned out that github-api and github-client were never actually used anywhere (at least I didn't get results with grep and nothing broke after I removed them). My guess is that when building Probo, people were testing different libraries and forgot to get rid of the unused ones.